.travis.deploy.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+cp .travis.settings.xml $HOME/.m2/settings.xml
```

Comment:
`set -e`
${HOME}

Reply:
${HOME} is in 90c7042438b8236c961cb14028b620139bbbc830
I believe Travis is going to do `set -e` for us in the [script deployment](https://docs.travis-ci.com/user/deployment/script/) which is a more specialized use case than scripts above. If they don't end up doing that, then I'll add it.